### PR TITLE
Max sebso id 19 global css forms, text, buttons

### PR DESCRIPTION
**Pull request by maxsebso**
I really tried my best but this css is so hard and I dont like it lmao. Fixed the landing page error and will continue to improve the css

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -203,4 +203,19 @@ th {
...
+
+/* SIGN IN/SIGN UP/ BASICALLY FORMS LMAO */
+form {
+  margin: auto;
+}
+
+form,
+button {
+  margin: 20px;
+}
```
Comment (wwohlers) [inline src/App.vue]: Overrides previous margin:auto in the form selector. To fix, remove the "form" in the "form, button" selector so that only buttons have margin:20px

File: src/App.vue [hunk 2]
```diff
@@ -203,4 +203,19 @@ th {
...
+
+form,
```
Comment (wwohlers) [inline src/App.vue]: Overrides previous margin:auto in the form selector. To fix, remove the "form" in the "form, label" selector so that only labels  have margin:20px

File: src/App.vue [hunk 3]
```diff
@@ -205,8 +205,21 @@ th {
...
 
 /* SIGN IN/SIGN UP/ BASICALLY FORMS LMAO */
-form {
+.container {
   margin: auto;
+  border: solid;
+  width: 450px;
+}
+
+form {
```
Comment (wwohlers) [inline src/App.vue]: Another form selector here...we need to make sure that there's only one general form selector.

File: src/App.vue [hunk 4]
```diff
@@ -205,8 +205,21 @@ th {
 }
 
 /* SIGN IN/SIGN UP/ BASICALLY FORMS LMAO */
-form {
+.container {
```
Comment (wwohlers) [inline src/App.vue]: Need to add as a comment where .container class is used, it's not clear.

File: src/App.vue [hunk 5]
```diff
@@ -205,8 +205,21 @@ th {
...
   margin: auto;
+  border: solid;
+  width: 450px;
+}
+
+form {
+  margin-left: auto;
+  margin-right: auto;
```
Comment (wwohlers) [inline src/App.vue]: replace with margin: 0 auto 0 auto;

File: src/views/Board.vue [hunk 6]
```diff
@@ -93,6 +93,10 @@ export default {
         }
       ]
     };
+  },
+  //makes it so that when taken to page from router, it goes to the top
```
Comment (wwohlers) [inline src/views/Board.vue]: hmm...why is this needed? This seems like an indirect solution to what might be a deeper-rooted problem

Comment (maxsebso) [reply src/views/Board.vue]: read my thread with jack

Comment (maxsebso) [reply src/views/Board.vue]: this is the solution that applicable, no deeper rooted problem

File: src/App.vue (hunk 4 above)
Comment (maxsebso) [reply src/App.vue]: This was all outdated code

Comment (maxsebso) [reply src/App.vue]: you can see that it says outdated on top

**Comment by maxsebso** [comment]
sorry I tired my best

File: src/views/Apply.vue [hunk 7]
```diff
@@ -12,7 +14,8 @@
         />
       </div>
       <div class="form-item">
-        <p>Year:</p>
+        Year
+        <br />
 
         <input type="radio" id="form-year-1" v-model="year" value="1" />
```
Comment (jackblanc) [inline src/views/Apply.vue]: Can we add more spacing between the buttons here?

File: src/views/Auth.vue [hunk 8]
```diff
@@ -1,11 +1,15 @@
 <template>
-  <div>
+  <div class="container">
+    <!-- TODO: MAKE TABS INSTEAD OF BUTTONS -->
```
Comment (jackblanc) [inline src/views/Auth.vue]: Can we copy the styling from https://dash.jackblanc.com/auth for toggling between sign in/sign up here? With the text below the two inputs.

File: src/views/Apply.vue (hunk 7 above)
Comment (jackblanc) [inline src/views/Apply.vue]: Can we add more spacing between the buttons here?

File: src/views/Board.vue (hunk 6 above)
Comment (jackblanc) [reply src/views/Board.vue]: lgtm

**Comment by jackblanc** [comment]
Oh also can we make the "Meet the Team" text on landing page the same size as "Get Involved Today!" and make the longer paragraph text throughout the page like 25% bigger.

File: src/views/Auth.vue (hunk 8 above)
Comment (maxsebso) [reply src/views/Auth.vue]: Sure I'll get on it asap

File: src/views/Apply.vue (hunk 7 above)
Comment (maxsebso) [reply src/views/Apply.vue]: sure

**Comment by maxsebso** [comment]
> Oh also can we make the "Meet the Team" text on landing page the same size as "Get Involved Today!" and make the longer paragraph text throughout the page like 25% bigger.

will do
